data-prepper-plugins/aggregate-processor/src/main/java/com/amazon/dataprepper/plugins/processor/aggregate/AggregateProcessorConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.aggregate;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.AssertTrue;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotEmpty;
+
+import java.io.File;
+import java.util.List;
+
+public class AggregateProcessorConfig {
+
+    static int DEFAULT_WINDOW_DURATION = 180;
+    static String DEFAULT_DB_PATH = "data/aggregate";
+
+    @JsonProperty("identification_keys")
```

Comment:
FYI, The plugin framework is configured to use snake_case when deserializing. Thus, this should work with only `@JsonProperty`. But, it is fine to keep the strings if you prefer.